source/index.txt

```diff
@@ -2,19 +2,129 @@
+security patches to them.
+
+.. card-group::
+   :columns: 3
+   :layout: default
+   :type: drivers
+
+   .. card::
+      :headline: C
+      :url: /c/
+      :icon: /figures/icons/c.svg
+      :icon-alt: C Driver icon
+
+   .. card::
+      :headline: C++
+      :url: /cxx/
+      :icon: /figures/icons/c++.svg
+      :icon-alt: C plus plus Driver icon
+
+   .. card::
+      :headline: C#
+      :url: /csharp/
+      :icon: /figures/icons/csharp.svg
+      :icon-alt: C sharp Driver icon
+
+   .. card::
+      :headline: Go
+      :url: https://www.mongodb.com/docs/drivers/go/current/
+      :icon: /figures/icons/go.svg
+      :icon-alt: Go Driver icon
+
+   .. card::
+      :headline: Java
+      :url: /java-drivers/
+      :icon: /figures/icons/java.svg
+      :icon-alt: Java Driver icon
+   
+   .. card::
+      :headline: Node.js
+      :url: https://www.mongodb.com/docs/drivers/node/current/
+      :icon: /figures/icons/node.svg
+      :icon-alt: Node Driver icon
+
+   .. card::
+      :headline: PHP
+      :url: /php/
+      :icon: /figures/icons/php.svg
+      :icon-alt: PHP Driver icon
+
+   .. card::
+      :headline: Python
+      :url: /python/
+      :icon: /figures/icons/python.svg
+      :icon-alt: Python Driver icon
+   
+   .. card::
+      :headline: Ruby
+      :url: https://www.mongodb.com/docs/ruby-driver/current/
+      :icon: /figures/icons/ruby.svg
+      :icon-alt: Ruby Driver icon
+
+   .. card::
+      :headline: Rust
+      :url: /rust/
+      :icon: /figures/icons/rust.svg
+      :icon-alt: Rust Driver icon
+
+   .. card::
+      :headline: Scala
+      :url: /scala/
+      :icon: /figures/icons/scala.svg
+      :icon-alt: Scala Driver icon
+
+   .. card::
+      :headline: Swift
+      :url: /swift/
+      :icon: /figures/icons/swift.svg
+      :icon-alt: Swift Driver icon
+
+   .. card::
+      :headline: TypeScript
+      :url: /typescript/
+      :icon: /figures/icons/ts.svg
+      :icon-alt: TypeScript Driver icon
+
+
+Featured Community Supported Libraries
+-----------------------------------------
+
+.. card-group::
+   :columns: 3
+   :layout: carousel
+   :type: drivers
+
+   .. card::
+      :cta: Elixir
+      :url: https://github.com/zookzook/elixir-mongodb-driver
+      :icon: /figures/icons/elixir.svg
+      :icon-alt: Elixir icon
+
+   .. card::
+      :cta: Kotlin
+      :url: https://github.com/Litote/kmongo
+      :icon: /figures/icons/kotlin.svg
+      :icon-alt: Kotlin icon
 
-The following libraries are officially supported by MongoDB. They are actively maintained, support new MongoDB features, and receive bug fixes, performance enhancements, and security patches.
+   .. card::
+      :cta: R
+      :url: https://cran.r-project.org/web/packages/mongolite/
+      :icon: /figures/icons/r.svg
+      :icon-alt: R icon
 
-.. drivers-index-tiles::
+Don’t see your desired language? Browse more `community supported libraries <https://github.com/ramnes/awesome-mongodb#libraries>`__.
```

Comment:
S: "favorite language" sounds a little more fun than "desired language". just my two cents

Reply:
I agree that "desired language" might sound very formal, but also think "favorite language" doesn't communicate the same message since I don't think the point is to find your favorite language, but rather the one you need to use due to any external constraints you have. 

Since this text is the same as what was there before, I'll leave it. I think Product will be asking for some updated copy soon.